translations/tiled_fr.ts

```diff
@@ -2595,7 +2596,7 @@ Voulez-vous le remplacer ?</translation>
     <message>
         <location line="-1223"/>
         <source>Ctrl+Shift+O</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+Màj+O</translation>
```

Comment:
Truly sorry for that one. I remember seeing several other shortcuts translated and thus thought the rule had changed. Thanks for fixing my mistakes. :)